wp-content/themes/citylimits/less/_navbar.less

```diff
@@ -98,6 +98,12 @@ html.nav-open body {
         }
     }
 }
+.sticky-nav-holder {
+    position: relative;
+    &.position-fixed {
+        position: fixed;
+    }
+}
```

Comment:
When this flips to the fixed position, the content on the page jumps up by the height of the sticky nav. Is there a way to avoid that by adding padding to a different element when `.position-fixed` is applied, or by keeping this element out of flow with `position: absolute` until it's `position: fixed`?

Reply:
> When this flips to the fixed position, the content on the page jumps up by the height of the sticky nav.

Could you explain that more? Not sure I understand :/

Reply:
I can't get this to reproduce now, so maybe it was fixed when you merged master in?

Before, when scrolling across the sticky/unsticky boundary, the headline was jumping up. Now it's just the width of the nav changing as it switches states. 

I can't get a gif of the width changing, so here's the pre-scroll and post-scroll appearances:

![Screen Shot 2019-10-28 at 17 36 01 ](https://user-images.githubusercontent.com/1754187/67720081-78069f80-f9a9-11e9-87b4-378a8bd6ff73.png)
![Screen Shot 2019-10-28 at 17 36 09 ](https://user-images.githubusercontent.com/1754187/67720082-789f3600-f9a9-11e9-95ce-636bfab5b84e.png)

Reply:
@benlk That was happening because the non-sticky version had a `padding: 0 2.5%` attribute that the sticky version did not have. https://github.com/INN/umbrella-citylimits/pull/81/commits/b9f9ce3e83ac034f092ac23a76e5e74e8e0344b4 fixes that